spi/data-plane/data-plane-spi/src/main/java/org/eclipse/edc/connector/dataplane/spi/resolver/DataPlaneDataAddressResolver.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2022 Amadeus
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Amadeus - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.dataplane.spi.resolver;
+
+import org.eclipse.edc.spi.result.Result;
+import org.eclipse.edc.spi.types.domain.DataAddress;
+
+/**
+ * Resolves the access token received in input of Data Plane public API into a {@link DataAddress}.
+ */
+@FunctionalInterface
+public interface DataPlaneDataAddressResolver {
```

Comment:
The interface could be called `DataAddressResolver` and the implementation `DataPlaneDataAddressResolver`, or, maybe better `ConsumerPullTransferDataAddressResolver`